tests/Findologic/Request/NavigationRequestFactoryTest.php

```diff
@@ -71,6 +74,8 @@ public function testNavigationRequestInstance(): void
         $this->assertSame($expectedIpAddress, $params['userip']);
         $this->assertSame($expectedAdapter, $params['outputAdapter']);
         $this->assertSame($expectedHost, $params['shopurl']);
+        $this->assertSame($expectedShopType, $params['shopType']);
+        $this->assertSame(Utils::getCleanShopwareVersion(), $params['shopVersion']);
```

Comment:
For consistency with other expected variables, add the Shopware version also in a variable above (similar to `$expectedShopType` and use that here. This is also fine, but its better to be consistent if making changes in an existing test so we can have all our expected variables in one place.